### DISCUSSION TITLE
Impossibility of linking draft issue in Github project

### DISCUSSION THREAD

**Discussion by msimic** [community/community#36163] [Projects and Issues]
You can create a URL link to share an issue that goes into the related repository.
You cannot create such a link for draft issues.
This makes Github projects borderline unusable for teams.
The one who creates the draft issue is not the one that will convert it to a real issue or reject it.
So collaboration between those two people is awful.

**Comment by nicosuave** [comment]
Agreed, this needs to change. Draft issues need an ID and subsequently, a link.

**Comment by mattcosta7** [comment]
https://github.blog/changelog/2023-01-05-github-issues-january-5th-update/

We recently shipped urls that link directly to the projects sidepanel open to an issue or draft issue, which should hopefully alleviate this for you
